cogboard-webapp/src/components/LastUpdate.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
-import {Typography} from "@material-ui/core";
+import styled from '@emotion/styled/macro';
```

Comment:
Empty line between libs imports and components imports.

Reply:
Done